src/machine/machine_atsamd51.go

```diff
@@ -2331,7 +2331,7 @@ func (dac DAC) Configure(config DACConfig) {
 	}
 
 	// enable
-	sam.DAC.CTRLB.Set(sam.DAC_CTRLB_REFSEL_VREFPU)
+	sam.DAC.CTRLB.Set(sam.DAC_CTRLB_REFSEL_VREFPU << sam.DAC_CTRLB_REFSEL_Pos)
```

Comment:
Register was in the wrong position, so I fixed it.
It does not work on VDDANA at this time.
It appears to be working with VREFPU or VREFPB.

Reply:
VREFPU
![image](https://user-images.githubusercontent.com/9251039/86578109-3efde080-bfb6-11ea-87c9-ec877ed55ca0.png)

VDDANA
I don't know why, but the VDDANA waveform looks like this
![image](https://user-images.githubusercontent.com/9251039/86578133-48874880-bfb6-11ea-81ad-c8978b519327.png)

Reply:
According to this post https://www.avrfreaks.net/comment/2620671#comment-2620671

> 1.8.2 DAC
> The selection of VDDANA as the DAC reference in DAC.CTRLB.REFSEL is non-functional.
> Workaround
> The VDDANA must be connected externally to a VREF pin and DAC.CTRLB.VREFAU must be selected.

Not sure why those spikes occur, but in any case seems like not expected to work anyhow.